### PR TITLE
Section1

### PR DESCRIPTION
**Pull request by MitskevichVS**
## Fixed: ##
 - dimentions in px;
- dynamical content slider;
- class names;
- block width dimentions;
- odd dimentions;
### Screenshot ###
screencapture-127-0-0-1-5500-dist-2020-01-13-16_13_45

### CODE REVIEW

File: dist/js/app/app.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+export default class App {
+    constructor() {
+        this.firstSliderControlsNode = document.querySelector('.slider__controls');;
+        this.firstSliderContentNode = document.querySelector('.content__slider');;
```
Comment (danymill) [inline dist/js/app/app.js]: double `;`

File: dist/js/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Slider from './slider/slider.js';
+
+window.onload = () => {
+    const slidersArray = Array.from(document.querySelectorAll('.section__content-slider'));
+    slidersArray.forEach((slider) => {
+        const sliderClass = new Slider(slider);
+        sliderClass.addSliderEventListeners();
```
Comment (danymill) [inline dist/js/index.js]: I would like to see more common `init` method :)

File: dist/js/slider/slider.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+export default class Slider {
+    constructor(element) {
+        this.firstSliderControlsNode = element.children[0];
```
Comment (danymill) [inline dist/js/slider/slider.js]: what the meaning of `first` word here?

File: dist/js/slider/slider.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        this.firstSliderContentNode = element;
+        this.firstSliderControls = Array.from(this.firstSliderControlsNode.children);
+        this.firstSliderContent = Array.from(this.firstSliderContentNode.children);
+    }
+
+    addSliderEventListeners() {
+        this.firstSliderControlsNode.addEventListener('click', (event) => {
+            if (!event.target.classList.contains('controls__control')) {
+                return;
+            }
```
Comment (danymill) [inline dist/js/slider/slider.js]: What will happen if an element with class `.controls__control` will have `span` inside and user clicks on this `span`?

File: dist/js/slider/slider.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+            const selectedIndex = this.firstSliderControls.indexOf(event.target);
+
+            this.firstSliderControls.forEach((item) => {
+                item.classList.remove('selected__control');
```
Comment (danymill) [inline dist/js/slider/slider.js]: Looks like the class `selected__control` is used as a modifier for some other class... let's rename it by starting with `_`

File: dist/js/slider/slider.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            });
+
+            event.target.classList.add('selected__control');
+
+            this.firstSliderContent.forEach((item, index) => {
+                item.classList.remove('selected__image');
+                if (index === selectedIndex + 1) {
```
Comment (danymill) [inline dist/js/slider/slider.js]: why `+` but not `-`? What am I missing?

Comment (MitskevichVS) [reply dist/js/slider/slider.js]: the first index is control container button, then first control match to the second child of sliderContentNode(first image)

File: dist/js/slider/slider.js (hunk 5 above)
Comment (MitskevichVS) [reply dist/js/slider/slider.js]: agree

File: dist/js/slider/slider.js (hunk 3 above)
Comment (MitskevichVS) [reply dist/js/slider/slider.js]: fixed

File: dist/js/slider/slider.js (hunk 5 above)
Comment (MitskevichVS) [reply dist/js/slider/slider.js]: fixed

File: dist/js/slider/slider.js (hunk 6 above)
Comment (MitskevichVS) [reply dist/js/slider/slider.js]: fixed

File: dist/js/slider/slider.js (hunk 4 above)
Comment (MitskevichVS) [reply dist/js/slider/slider.js]: fixed

File: dist/js/index.js (hunk 2 above)
Comment (MitskevichVS) [reply dist/js/index.js]: fixed

**Pull request merged**